require that `$schema` cannot contain a fragment (#1633)
json-schema-org/json-schema-spec: specs/jsonschema-core.md

including the `$schema` keyword with a different value.

The value of this keyword MUST be an
[absolute IRI](https://www.rfc-editor.org/info/rfc3987) (without a fragment). This

> IRI MUST be normalized.

Author: That language is actually in the first paragraph of the very next section.  I've added a link.

including the `$schema` keyword with a different value.

The value of this keyword MUST be an

> [IRI](https://www.rfc-editor.org/info/rfc3987) (containing a scheme). This
> IRI MUST be normalized and MUST NOT contain a fragment.

Reviewer: This can be called an "absolute IRI" instead of just an "IRI". Then we don't have to make a normative statement that a fragment isn't allowed because that's part of the definition of "absolute IRI". However, it might be a good idea to clarify that fact since most people think "absolute" just means "non-relative". Which means we shouldn't need to clarify that it mean "containing a scheme" anymore. Maybe something like this would work,

```suggestion
[absolute IRI](https://www.rfc-editor.org/info/rfc3987) (without a fragment). This
IRI MUST be normalized.
```
